src/components/MAPI/apps/ClusterDetailAppListItem.tsx

```diff
@@ -37,15 +44,40 @@ const ClusterDetailAppListItem: React.FC<IClusterDetailAppListItemProps> = ({
     ? applicationv1alpha1.getAppCurrentVersion(app)
     : undefined;
 
+  const isDeleted = typeof app?.metadata?.deletionTimestamp !== 'undefined';
+  const isDisabled = typeof app === 'undefined' || isDeleted;
+
+  const accordionRef = useRef<HTMLDivElement>(null);
+
+  useLayoutEffect(() => {
+    if (!accordionRef) return;
+
+    const accordionButton = accordionRef.current?.querySelector('button');
+    if (!accordionButton) return;
+
+    accordionButton.disabled = isDisabled;
+  }, [isDisabled]);
+
+  const handleHeaderClick = (e: React.MouseEvent<HTMLElement>) => {
+    if (isDisabled) {
```

Comment:
Blocking click events if the app is deleting, so the user can not open the accordion